src/components/reusable/codeView/codeView.stories.js

```diff
@@ -0,0 +1,242 @@
+import { html } from 'lit';
+import { action } from '@storybook/addon-actions';
+import './index';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+
+const CODE_VIEW_TYPES = {
+  INLINE: 'inline',
+  BLOCK: 'block',
+};
+
+const removeLeadingWhitespace = (code) => {
+  if (!code) return '';
+  const lines = code.split('\n');
+  const minIndent = lines.reduce((min, line) => {
+    const indent = line.match(/^\s*/)[0].length;
+    return line.trim().length ? Math.min(min, indent) : min;
+  }, Infinity);
+  return lines
+    .map((line) => line.slice(minIndent))
+    .join('\n')
+    .trim();
+};
+
+export default {
+  title: 'Components/CodeView',
+  component: 'kyn-code-view',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: '',
+    },
+  },
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg', 'auto'],
+      control: { type: 'select' },
+    },
+    language: {
+      options: [
+        'javascript',
+        'html',
+        'css',
+        'scss',
+        'json',
+        'xml',
+        'markdown',
+        'svg',
+        'yaml',
+        'bash',
+      ],
+      control: { type: 'select' },
+    },
+    copyOptionVisible: {
+      control: { type: 'boolean' },
+    },
+  },
+};
+
+const Template = (args) => {
+  if (args.snippetType === CODE_VIEW_TYPES.INLINE) {
+    return html`
+      <kyn-code-view
+        title=${args.title}
+        snippetType=${args.snippetType}
+        language=${args.language}
+        ?copyOptionVisible=${false}
+        copyButtonText=${''}
+        @on-custom-copy=${(e) => action('on-custom-copy')(e.detail)}
+      >
+        <span slot="inline-example">${args.exampleInlinetext}</span>
+        <pre><code>${removeLeadingWhitespace(args.code)}</code></pre>
```

Comment:
from what i'm reading, it's better to wrap with `<pre>` and `<code>` on the storybook side, due to shadow DOM limitations and styling control, so i've going to remove the redundancy of that wrapper on the component side for both the inline and block implementations.

you're way more versed in these things though, so if you think it's better to try and have it on the component side, i'm down

Reply:
this has been refactored. i'm also defaulting to the darker background and the `tomorrow` theme, with a slight modification to two hex values so that the accessibility contrast tests would pass. 

if another one is preferred or if the shidoka theme materializes, i'm happy to swap it out.